BeeSwiftTests/GoalViewModelTests.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  GoalViewModelTests.swift
+//  BeeSwiftTests
+//
+//  Created by krugerk on 2024-11-25.
+//
+
+import Testing
+
+@testable import BeeSwift
+@testable import BeeKit
+
+struct GoalViewModelTests {
+    
+    @Test func initialStepperIsMinusOneWhenSubmissionDateIsAfterMidnightAndBeforeDeadline() async throws {
```

Comment:
This is the new SwiftTesting syntax right? 😍

Reply:
and more such migration #530